HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestDetailsPageController.java

```diff
@@ -608,6 +608,12 @@ public ModelAndView initPage(final ReferenceManifestDetailsPageParams params,
                 String uuidError = "Failed to parse ID from: " + params.getId();
                 messages.addError(uuidError);
                 log.error(uuidError, iaEx);
+            } catch (CertificateException cEx) {
```

Comment:
I noticed that we are catching every possible exception the application could throw at us (with the very last catch statement on line 617 in the modified file), is that last catch necessary? If it is necessary, do you believe it would make sense to collapse all of the above exceptions into one big catch seeing as that we are logging the same exception and we aren't doing anything special per exception?

Reply:
Exception handling can be more elegantly handled in the ACA.  In many cases a try block is followed by several catches with unique parameters that are otherwise identical.  If deemed appropriate these can be addressed in a future PR.